docs/source/tutorials_source/platform/tutorial_active_learning_detectron2.py

```diff
@@ -0,0 +1,306 @@
+*   how to download the selected samples for labeling
+
+The tutorial will be divided into
+the following steps. 
+
+#. Installation of detectron2 and lightly
+#. Run predictions using a pretrained model
+#. Use lightly to compute active learning scores for the predictions
+#. Use the Lightly Platform to understand where our model struggles
+#. Select the most valuable 100 images for annotation
+
+Requirements
+------------
+- Make sure you have the detectron2 framework installed on your machine. Check out
+  the `detectron2 installation documentation <https://detectron2.readthedocs.io/en/latest/tutorials/install.html>`_
+
+- In this tutorial we work with the comma10k dataset. The dataset consists of
+  10'000 images for autonomous driving and is available
+  `here on GitHub <https://github.com/commaai/comma10k>`_
+  We can download the dataset using `git clone`. We save the dataset locally
+  to `/datasets/`
+
+.. code::
+
+    git clone https://github.com/commaai/comma10k
+
+"""
+
+# Some basic setup:
+# Setup detectron2 logger
+import detectron2
+from detectron2.utils.logger import setup_logger
+setup_logger()
+
+# import some common libraries
+import numpy as np
+import os, json, cv2, random, glob
+import tqdm, gc
+import matplotlib.pyplot as plt
+
+# import some common detectron2 utilities
+from detectron2 import model_zoo
+from detectron2.engine import DefaultPredictor
+from detectron2.config import get_cfg
+from detectron2.utils.visualizer import Visualizer
+from detectron2.data import MetadataCatalog, DatasetCatalog
+
+# includes for lightly
+import lightly
+from lightly.active_learning.utils.bounding_box import BoundingBox
+from lightly.active_learning.utils.object_detection_output import ObjectDetectionOutput
+from lightly.active_learning.scorers import ScorerObjectDetection
+from lightly.api.api_workflow_client import ApiWorkflowClient
+from lightly.active_learning.agents import ActiveLearningAgent
+from lightly.active_learning.config import SamplerConfig
+from lightly.openapi_generated.swagger_client import SamplingMethod
+
+# %%
+# Upload dataset to lightly
+# -------------------------
+#
+# To work with the Lightly Platform and use the active learning feature we
+# need to upload the dataset. 
+# 
+# First, head over to `the Lightly Platform <https://app.lightly.ai/>`_ and 
+# create a new dataset.
+#
+# We can now upload the data using using the command line interface. Replace 
+# **yourToken** and *yourDatasetId** with the two provided values from the web app.
+# Don't forget to adjust the **input_dir** to the location of your dataset.
+#
+# .. code:: 
+# 
+#     lightly-magic token="yourToken" dataset_id="yourDatasetId" \
+#         input_dir='/datasets/comma10k/imgs/' trainer.max_epochs=20 \
+#         loader.batch_size=64 loader.num_workers=2
+#
+# .. note::
+#
+#     In this tutorial we use the lightly-magic command which trains a model
+#     before embedding and uploading it to the Lightly Platform.
```

Comment:
Can you add something like "To skip training, you can set `trainer.max_epochs=0`"